raiden/raiden_service.py

```diff
@@ -647,14 +1070,17 @@ def __init__(self, raiden):
 
     def on_message(self, message, msghash):  # noqa pylint: disable=unused-argument
         """ Handles `message` and sends an ACK on success. """
+        if log.isEnabledFor(logging.INFO):
+            log.info('message received', message=message)
+
         cmdid = message.cmdid
 
         # using explicity dispatch to make the code grepable
         if cmdid == messages.ACK:
             pass
 
         elif cmdid == messages.PING:
-            self.message_ping(message)
+            pass
```

Comment:
I see. Yeah I expected us to send an ACK to a ping so this was a bit confusing. Thanks for the explanation.